### DISCUSSION TITLE
Cache miss/hit and access to page numbers

### DISCUSSION THREAD

**Discussion by komeilYahyazade** [sarchlab/mgpusim#63] [Q&A] [answered]
Hello author, i'm trying to obtain what pages are accessed and see if cache miss/hit in each access but i can't find a way in order to do this.
can you please give me some hints(like what functions are involved)?

**Comment by syifan** [comment, marked as answer]
There is an option called `-trace-mem`. It will generate traces like this:

```
start, 0.000083854000, GPU[1].L2[7], 436721@GPU[1].L2[7], *mem.WriteReq, 0x1000063c0, 64
```

The second field from the end is the address. You can use this trace to examine what pages are accessed at each level of caches and DRAMs.

**Reply by Yzgll** [reply]
Mr. Sun, I also have a discussion, which is the instruction code I added. I hope you can point out the mistakes I made.
